content/influxdb/cloud/write-data/troubleshoot.md

```diff
@@ -18,45 +18,48 @@ related:
 
 Learn how to handle and recover from errors when writing to InfluxDB.
 
-- [Discover common failure scenarios](#common-failure-scenarios)
+- [Discover common failure scenarios](#discover-common-failure-scenarios)
 - [Review HTTP status codes](#review-http-status-codes)
 - [Troubleshoot failures](#troubleshoot-failures)
 
-## Common failure scenarios
+## Discover common failure scenarios
 
 InfluxDB write requests may fail for a number of reasons.
 Common failure scenarios that return an HTTP `4xx` or `5xx` error status code include the following:
 
 - Exceeded a rate limit.
 - API token was invalid.
 - Client or server reached a timeout threshold.
-- Size of the data payload was too large.
 - Data was not formatted correctly.
 - Data did not conform to the [explicit bucket schema](/influxdb/cloud/organizations/buckets/bucket-schema/).
   See how to troubleshoot specific [bucket schema errors](/influxdb/cloud/organizations/buckets/bucket-schema/#troubleshoot-errors).
+- Size of the data payload was too large.
```

Comment:
@jstirnaman we're working on a new [InfluxDB Cloud limits doc](https://github.com/influxdata/docs-v2/pull/3355/files). Ideally, it'd be nice to link to this new doc with: "To learn more about InfluxDB Cloud limits, see..." prob for another issue/PR, unless that PR goes out first

Reply:
may be more detail than in scope here? adding in case it makes sense to add within this context:

per @rickspencer3 [PR](https://github.com/influxdata/docs-v2/pull/3392/files), we may want to note:
_After 250 MB of data from a request payload is decompressed and written to InfluxDB, the remaining lines in the payload are not written. We recommend retrying the request with a smaller payload._

Reply:
```suggestion
```